src/objectTranslations/Contact-ca.objectTranslation

```diff
@@ -3481,11 +3485,11 @@
     <label>Sexe</label>
     <picklistValues>
       <masterLabel>Female</masterLabel>
-      <translation>Dona</translation>
+      <translation>Female</translation>
     </picklistValues>
     <picklistValues>
       <masterLabel>Male</masterLabel>
-      <translation>Home</translation>
+      <translation>Male</translation>
```

Comment:
These values should remain translated. Only the specific custom label should have had its translation reversed.
```suggestion
      <translation>Home</translation>
```